CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## v0.1.1 (4th July 2023)
```

Comment:
Would prefer 2023-07-04 format to match other repos

Reply:
i knew you would say that 🤣 
Will change it 👍